src/LiveComponent/src/Metadata/LiveComponentMetadataFactory.php

```diff
@@ -71,6 +71,9 @@ public function createPropMetadatas(\ReflectionClass $class): array
             }
 
             $type = $property->getType();
+            if ($type instanceof \ReflectionUnionType || $type instanceof \ReflectionIntersectionType) {
+                throw new \LogicException(sprintf('Union nor intersection types are not supported for LiveProps. You may want to change the type of property %s in %s.', $property->getName(), $property->getDeclaringClass()->getName()));
```

Comment:
```suggestion
                throw new \LogicException(sprintf('Union or intersection types are not supported for LiveProps. You may want to change the type of property %s in %s.', $property->getName(), $property->getDeclaringClass()->getName()));
```

Just to avoid the double-negative of `nor` and `not supported`.